bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/handlers/ShowViewHandler.java

```diff
@@ -55,14 +55,9 @@ public ShowViewHandler() {
 	}
 
 	/**
-	 * Creates a new ShowViewHandler that will optionally force the view to become a
-	 * fast view.
-	 *
-	 * @param makeFast if true, the view will be moved to the fast view bar (even if
-	 *                 it already exists elsewhere). If false, the view will be
-	 *                 shown in its default location. Calling with false is
-	 *                 equivalent to using the default constructor.
+	 * fast views are not supported since a while in the Eclipse IDE
 	 */
+	@Deprecated(forRemoval = true)
```

Comment:
add since 2023-12

Reply:
Shouldn’t it use a version for the bundle?